libs/ml/benchmark/activations/activations.cpp

```diff
@@ -57,6 +57,37 @@ BENCHMARK_TEMPLATE(BM_DropoutForward, double, 1024)->Unit(benchmark::kMicrosecon
 BENCHMARK_TEMPLATE(BM_DropoutForward, double, 2048)->Unit(benchmark::kMicrosecond);
 BENCHMARK_TEMPLATE(BM_DropoutForward, double, 4096)->Unit(benchmark::kMillisecond);
 
+template <class T, int N>
+void BM_DropoutBackward(benchmark::State &state)
+{
+  using TensorType  = typename fetch::math::Tensor<T>;
+  auto input        = TensorType({1, N});
+  auto output       = TensorType({1, N});
+  auto error_signal = TensorType({1, N});
+
+  // Fill tensors with random values
+  input.FillUniformRandom();
+  error_signal.FillUniformRandom();
+
+  std::vector<std::shared_ptr<fetch::math::Tensor<T> const>> inputs;
+
+  inputs.emplace_back(std::make_shared<TensorType>(input));
+  fetch::ml::ops::Dropout<fetch::math::Tensor<T>> dm(0.5);
+  dm.Forward(inputs, output);
+
+  for (auto _ : state)
+  {
+    dm.Backward(inputs, error_signal);
+  }
+}
+
+BENCHMARK_TEMPLATE(BM_DropoutBackward, double, 2)->Unit(benchmark::kNanosecond);
+BENCHMARK_TEMPLATE(BM_DropoutBackward, double, 256)->Unit(benchmark::kMicrosecond);
+BENCHMARK_TEMPLATE(BM_DropoutBackward, double, 512)->Unit(benchmark::kMicrosecond);
+BENCHMARK_TEMPLATE(BM_DropoutBackward, double, 1024)->Unit(benchmark::kMicrosecond);
+BENCHMARK_TEMPLATE(BM_DropoutBackward, double, 2048)->Unit(benchmark::kMicrosecond);
+BENCHMARK_TEMPLATE(BM_DropoutBackward, double, 4096)->Unit(benchmark::kMillisecond);
```

Comment:
I ran these locally, and 4096 runs in 0ms. Can you adjust the units so that the results are between say 10 and 10000 for the selected unit?